optimize/backend/src/it/java/io/camunda/optimize/test/it/extension/db/ElasticsearchDatabaseTestService.java

```diff
@@ -413,12 +413,7 @@ public void updateZeebeRecordsForPrefix(
                       u.index(zeebeRecordPrefix + "_" + indexName + "*")
                           .refresh(true)
                           .script(
-                              Script.of(
-                                  s ->
-                                      s.inline(
-                                          i ->
-                                              i.lang(ScriptLanguage.Painless)
-                                                  .source(updateScript))))
+                              Script.of(s -> s.lang(ScriptLanguage.Painless).source(updateScript)))
```

Comment:
Out of curiosity because this sort of API migration is something I may face during this migration, how do you know that you need remove `s.inline...` code here? Is there a migration guide?